apps/demo/package.json

```diff
@@ -25,7 +25,7 @@
 		"@nativescript/imagepicker": "file:../../packages/imagepicker",
 		"@nativescript/iqkeyboardmanager": "file:../../packages/iqkeyboardmanager",
 		"@nativescript/local-notifications": "file:../../packages/local-notifications",
-		"@nativescript/localize": "file:../../dist/packages/localize",
+		"@nativescript/localize": "file:../../packages/localize",
```

Comment:
This change here I believe will cause issue. `localize` right now is the only *exception* in the repo because it contains `hooks`. The hooks have to be built in order to process in the demo, thus `localize` needs to be built into dist prior to usage and why this should reference the `dist` location. Eventually we'd like to find a way to allow hooks to be live compiled with the other source but they currently have to be compiled to es5. A larger story would be to revise hooks altogether and just process them purely via `nativescript.config.ts` instead of needing a whole folder setup.